### ISSUE TITLE
ui: benchmark result view: wrap all list fields (& re-join lists of text lines?)

### ISSUE THREAD

**Issue by alistaire47** [conbench/conbench#780]
Sorry I don't have an example link (this is on a local instance), but when I have a list of strings in `error.log`, it doesn't wrap even though it's very very wide:

<img width="577" alt="image" src="https://user-images.githubusercontent.com/1693477/221689614-f9b1df05-35e4-419e-95cf-8879b7c637c7.png">

However, fields in `optional_benchmark_details` for the same result do wrap:

<img width="577" alt="image" src="https://user-images.githubusercontent.com/1693477/221690069-b1aa01e4-e836-4239-9b47-43493ce752f1.png">

Ideally for readability we'd probably reindent these and add newlines between strings (these are logs or scripts split into lines), but at a minimum wrapping would be nice.

**Comment by jgehrcke** [comment]
Absolutely. Thanks for adding this here @alistaire47 -- we should totally keep track of these ideas / small tasks in order to make this kind of work plannable.
